packages/liveblocks-node/src/client.ts

```diff
@@ -194,6 +273,518 @@ export class Liveblocks {
+   */
+  public getStorageDocument(
+    roomId: string,
+    format: "plain-lson"
+  ): Promise<PlainLsonObject>;
+
+  public getStorageDocument(roomId: string): Promise<PlainLsonObject>; // Default to 'plain-lson' when no format is provided
+
+  public getStorageDocument(
+    roomId: string,
+    format: "json"
+  ): Promise<JsonObject>;
+
+  public async getStorageDocument(
+    roomId: string,
+    format: "plain-lson" | "json" = "plain-lson"
+  ): Promise<PlainLsonObject | JsonObject> {
+    let path = `v2/rooms/${roomId}/storage`;
+    if (format === "json") {
+      path += "?format=json";
+    }
+    const res = await this.get(path);
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return res.json() as Promise<PlainLsonObject | JsonObject>;
+  }
+
+  /**
+   * Initializes a room’s Storage. The room must already exist and have an empty Storage.
+   * Calling this endpoint will disconnect all users from the room if there are any.
+   *
+   * @param roomId The id of the room to initialize the storage from.
+   * @param document The document to initialize the storage with.
+   * @returns The initialized storage document. It is of the same format as the one passed in.
+   */
+  public async initializeStorageDocument(
+    roomId: string,
+    document: PlainLsonObject
+  ): Promise<PlainLsonObject> {
+    const res = await this.post(`/v2/rooms/${roomId}/storage`, document);
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return res.json() as Promise<PlainLsonObject>;
+  }
+  /**
+   * Deletes all of the room’s Storage data and disconnect all users from the room if there are any.
+   * @param roomId The id of the room to delete the storage from.
+   */
+  public async deleteStorage(roomId: string): Promise<void> {
+    const res = await this.delete(`/v2/rooms/${roomId}/storage`);
+
+    if (!res.ok) {
+      const text = await res.text();
+      throw new ApiError(res.status, text);
+    }
+
+    return;
+  }
+
+  /* -------------------------------------------------------------------------------------------------
+   * Yjs
+   * -----------------------------------------------------------------------------------------------*/
+
+  /**
+   * Returns a JSON representation of the room’s Yjs document.
+   * @param roomId The id of the room to get the Yjs document from.
+   * @param params.format (optional) If true, YText will return formatting.
+   * @param params.key (optional) If provided, returns only a single key’s value, e.g. doc.get(key).toJSON().
+   * @param params.type (optional) Used with key to override the inferred type, i.e. "ymap" will return doc.get(key, Y.Map).
+   * @returns A JSON representation of the room’s Yjs document.
+   */
+  public async getYjsDocument(
+    roomId: string,
+    params: {
+      format?: boolean;
+      key?: string;
+      type?: string;
```

Comment:
@jrowny The [REST documentation on Yjs](https://liveblocks.io/docs/api-reference/rest-api-endpoints#get-rooms-roomId-ydoc) mentions only `formatting`, `key`, and `type` as accepted parameters but the backend seems to also accept `guid`. I am guessing `guid` is related to sub-documents. Wondering if you know `guid` will also be documented in the REST documentation soon and if we want to include it here after the addition.

Same question for the `sendYjsBinaryUpdate` function ([REST documentation](https://liveblocks.io/docs/api-reference/rest-api-endpoints#put-rooms-roomId-ydoc))

Reply:
yes, guid is a new option that I did not document yet, all it does is specify which subdoc to load. I will add that rest documentation soon.